draft-ietf-tls-tls13.md

```diff
@@ -3340,14 +3340,14 @@ provides several recommendations to assist implementors.
 
 ## Random Number Generation and Seeding
 
-TLS requires a cryptographically secure pseudorandom number generator (PRNG).
+TLS requires a cryptographically secure pseudo-random number generator (PRNG).
 Care must be taken in designing and seeding PRNGs. PRNGs based on secure hash
 operations, most notably SHA-1, are acceptable, but cannot provide more
 security than the size of the random number generator state.
 
 To estimate the amount of seed material being produced, add the number of bits
 of unpredictable information in each seed byte. For example, keystroke timing
-values taken from a PC compatible's 18.2 Hz timer provide 1 or 2 secure bits
+values taken from a PC's compatible 18.2 Hz timer provide 1 or 2 secure bits
```

Comment:
This one is a bit different.

"PC compatible's 18.2 Hz timer" is wrong, though it might be fixable if a hyphen were stuck in the middle of "PC compatible".

"PC's compatible 18.2 Hz timer" is wrong, as it changes the meaning. "(IBM) PC compatible" is a specific term.
https://en.wikipedia.org/wiki/IBM_PC_compatible

I think the correct solution is to simply drop the "'s":
"values taken from a PC compatible 18.2 Hz timer provide 1 or 2 secure bits"

The word "timer" is the noun here, and all preceding are adjectives. The timer is "18.2 Hz" and is "PC compatible". We don't need the possessive form here.